src/borg/remote.py

```diff
@@ -21,6 +22,7 @@
 from .repository import Repository
 
 RPC_PROTOCOL_VERSION = 2
+BORG_VERSION = (1, 1, 0)
```

Comment:
hmm, this looks like a problem. our usual version is automatically maintained by setuptools-scm (and way more complex than this). mainting a separate tuple here seems not desirable.